src/chain/chain.rs

```diff
@@ -864,8 +864,8 @@ impl Chain {
     }
 
     /// Return the keys we know
```

Comment:
Would it make things easier for this change if we did the work of making relocate like GenesisUpdate i.e don't have to sign it before it goes through Parsec? then we can just add whatever we need when we send the message.